### DISCUSSION TITLE
Apply standard formatting to fields in labels

### DISCUSSION THREAD

**Discussion by MattIPv4** [marcusolsson/grafana-hexmap-panel#19] [Ideas]
👋 I love that the plugin lets you display multiple fields so cleaning as part of the tooltip label that shows -- however, it'd be awesome to see this augmented slightly further to support Grafana's standard unit formatting options, so that I could select a format for each field being displayed in the label (e.g. I have a timestamp column and a percentage, format both appropriately)

**Comment by marcusolsson** [comment]
I think this is a good idea! I'm not sure when I'll be able to work on this, but I'd be happy to accept a pull request!
